.gitignore

```diff
@@ -10,3 +10,6 @@ coverage
 deployment/*.retry
 
 /yarn-error.log
+
+test/image/build
+test/image/diff
```

Comment:
`test/image/build` and `test/image/diff` aren't used anymore after https://github.com/plotly/orca/commit/697068a13a470c9b74d644f255fb2f72eeeddeed , correct?

Reply:
Correct! Reverted `.gitignore` in 36bb04f6275617737b2ce21645266ce0e53eb300

Reply:
:tada: